andresSaldanaAguilar/react-certification-2021
# Intro to testing

#### What does this PR do?

Fulfills the tasks for the Mini-Challenge 2: Intro to Testing

#### How should this be manually tested?

You can open the Codesandbox pointing to this branch and you will able to see the app running, also I added the coverage output at the root, it is named [coverage.txt](https://github.com/andresSaldanaAguilar/react-certification-2021/blob/Intro-to-Testing/coverage.txt)

#### Checklist

- [x] I added tests for all my components and views using **jest and react testing library**
- [x] I removed some dead code and commented code that could be useful later for the course
